docs/test-data/Zone/Zone09.md

```diff
@@ -0,0 +1,132 @@
+# Specification of test data for ZONE09
+
+
+## Test Case
+This document specifies available test data for test case [Zone09].
+
+
+## Test scenarios
+
+The purpose of the test scenarios is to cover all reasonable contexts where
+different message tags are outputted when [Zone09] is run on a test zone. The
+message tags are defined in the test case ([Zone09]) and the scenarios are
+defined below.
+
+The scenarios are defined in two parts:
+
+* What messages from [Zone09] are expected to be outputted and what messages
+  are expected not to be outputted when a test zone setup according to the
+  scenario is tested by the test case.
+* Specification of the zone setup for the scenario.
+
+
+## Test environment
+
+The tests are assumed to be run in a closed environment with a private DNS tree
+to achieve full access to any zone. Configuration data and instructions to set
+this up are available elsewhere in this repository.
+
+The base domain name for the test zones in this document is assumed to be
+`zone09.xa` and the test domains are assumed to be the scenario name as a
+subdomain to the base name, e.g. `NO-RESPONSE-MX-QUERY.zone09.xa` except for a
+few cases where the test zone must be the root, a TLD or a domain name under
+.ARPA. Other test domains can be used, if desired.
```

Comment:
I don't quite understand the usefulness of the sentence “Other test domains can be used, if desired”. Shouldn’t we standardize on the `<scenario name>.<test case name>.xa` layout as much as possible?

Maybe it’s best to state that other test domains are only to be used if, for some reason, using the standard zone names (`<scenario name>.<test case name>.xa`) wouldn’t work. In other words, replacing “desired” with “necessary”?

Reply:
It is a good comment. From a unit test perspective the domain names must be fixed. For that we have decided to use the `<scenario name>.<test case name>.xa` pattern unless it is not possible (where the test zone must be a TLD, ARPA or root zone). The "if desired" is meant for another application. I will reword to avoid confusion.